<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ProcessPage</class>
    <widget class="QWidget" name="ProcessPage">
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="statusLabel">
                    <property name="text">
                        <string>Idle...</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QProgressBar" name="progressBar">
                    <property name="minimum">
                        <number>0</number>
                    </property>
                    <property name="maximum">
                        <number>100</number>
                    </property>
                    <property name="value">
                        <number>0</number>
                    </property>
                    <property name="textVisible">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QTextEdit" name="logTextEdit">
                    <property name="readOnly">
                        <bool>true</bool>
                    </property>
                    <property name="placeholderText">
                        <string>Log output will appear here...</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="cancelButton">
                    <property name="text">
                        <string>Cancel</string>
                    </property>
                    <property name="enabled">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>